src/deployment/README.md

```diff
@@ -101,7 +102,7 @@ Start a node by running:
 npx hardhat node
 ```
 
-Prefill deploy_parameters where possible
+Prefill deploy_parameters where possible. See docs/deployment_params.txt for details.
```

Comment:
probably its easiert to link to the official documentaiton than our own maintained thing:

https://docs.polygon.technology/zkEVM/get-started/deploy-zkevm/create-wallets/#prepare-deploy-configuration

Reply:
Not much of what I've commented is in the official documentation and we're doing a lot of stuff that's non-obvious, like 1_deployForkableToken.js is writing to deploy_parameters.json